pytorch_lightning/trainer/trainer.py

```diff
@@ -505,13 +505,15 @@ def _call_and_handle_interrupt(self, trainer_fn: Callable, *args: Any, **kwargs:
         """
         try:
             return trainer_fn(*args, **kwargs)
+        # TODO(@daniellepintz): treat KeyboardInterrupt as BaseException (delete the code below) in v1.7
```

Comment:
not sure about the convention around github names in the codebase. so far we've left todos without them. @Borda is there a style guide we ought to file for todos?

Reply:
I saw quite a few TODOs with the github names, so figured I would add mine as well so I can easily search for all of my todos in the future. happy to change though if that's not what we want

Reply:
I would not rather have names with todos, all todos shall be transferable so anyone can take it, meaning add enough information instead of writing your name (my personal experience is that I won't remember in a few weeks anyway :])

Reply:
sg!